include/drivers/clock_control/nrf_clock_control.h

```diff
@@ -28,6 +30,13 @@ enum clock_control_nrf_type {
 #define CLOCK_CONTROL_NRF_SUBSYS_LF \
 	((clock_control_subsys_t)CLOCK_CONTROL_NRF_TYPE_LFCLK)
 
+/** @brief LF clock start modes. */
+enum nrf_lfclk_start_mode {
+	NRF_LFCLK_START_MODE_NOWAIT,
+	NRF_LFCLK_START_MODE_SPINWAIT_STABLE,
```

Comment:
If RUNNING occurs before STABLE it would be nice to have that earlier in the enumeration, so magnitude corresponds to functionality.

Reply:
done